packages/react-native/template/metro.config.js

```diff
@@ -1,11 +1,32 @@
-const {getDefaultConfig, mergeConfig} = require('@react-native/metro-config');
+const {getDefaultConfig} = require('@react-native/metro-config');
 
 /**
  * Metro configuration
  * https://facebook.github.io/metro/docs/configuration
  *
  * @type {import('metro-config').MetroConfig}
  */
-const config = {};
+const config = getDefaultConfig(__dirname);
 
-module.exports = mergeConfig(getDefaultConfig(__dirname), config);
+// When enabled, the optional code below will allow Metro to resolve
+// and bundle source files with TV-specific extensions
+// (e.g., *.ios.tv.tsx, *.android.tv.tsx, *.tv.tsx)
+//
+// Metro will still resolve source files with standard extensions
+// as usual if TV-specific files are not found for a module.
+//
+// This code is not enabled by default, since it will impact bundling performance,
+// but is available for developers who need this capability.
+//
+/*
+if (process.env.BUILDING_FOR_TV) {
```

Comment:
One more question: So users should set this env variable by them selves or is it already set?

Reply:
Does this mean you couldn’t have parallel builds running in the same shell environment? Is there a way to pass this is in CLI or react-native.config.js in the specific app subdir, rather than a mutable global environment variable?

Reply:
@matthargett Yeah, sadly. We need extra Metro capabilities to make this smarter.

Reply:
@matthargett you could modify the scripts in the Xcode project build phases for starting packager and for bundling. Add the new env variable there for the TV target but not for the iOS target.

Reply:
Won’t this make project conversion and docs work more difficult later? Can we send the needed PR to metro and/or CLI (and/or repack), like @okwasniewski gas been doing to enable visionOS? Is there an incontrovertibly pressing business need to deploy this as quickly as possible with this major caveat?